apps/web/playwright/bookings-webhooks.e2e.ts

```diff
@@ -0,0 +1,233 @@
+import { expect } from "@playwright/test";
```

Comment:
I feel like webhook.e2e.ts is the right place for these tests. That file verifies settings webhooks flow and it verifies the event-type webhook flow.

They share a lot of logic as well.